react-app-drawer-component/dist/styles.css

```diff
@@ -0,0 +1,79 @@
+body {
+  font-family: sans-serif;
+  margin: 0;
+}
+
+i {
+  font-size: 2rem;
+  margin: 10px 10px;
+}
+
+i:hover {
+  cursor: pointer;
+}
+
+img {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.nav-menu {
+  padding: 1rem 0;
+  height: 100%;
+  width: 0;
+  overflow-x: hidden;
+  box-shadow: 4px 4px 5px rgba(0, 0, 0, 0.25);
+  white-space: pre;
+  transition: width 0.25s;
```

Comment:
Instead of transitioning the width, try a translate transformation. Transitioning width usually leads to document reflows and is less efficient. Also, it can look weird as content of the box reorganizes as the width grows or shrinks.